makefiles/mcuboot.mk

```diff
@@ -1,49 +1,52 @@
+.PHONY: mcuboot mcuboot-create-key mcuboot-flash-bootloader mcuboot-flash
+
 ifdef MCUBOOT_SLOT0_SIZE
 
 IMGTOOL ?= $(RIOTTOOLS)/mcuboot/imgtool.py
-override IMGTOOL := $(abspath $(IMGTOOL))
+IMGTOOL_KEYFLAGS ?= keygen -t rsa-2048
```

Comment:
Currently I do not find necessary to add the `KEYFLAGS`. If it was `imgtool.py` specific I would not put `keygen` in it.
And if supposed to be generic, here you do not include the `-k` so cannot be replaced by an other tool anyway.

Maybe keep this change for when there is a need.